CONTRIBUTING.md

```diff
@@ -31,11 +31,11 @@ source .tox/unit/bin/activate
 ### Testing
 
 ```shell
-tox -e fmt           # update your code according to linting rules
-tox -e lint          # code style
-tox -e unit          # unit tests
-tox -e integration   # integration tests
-tox                  # runs 'lint' and 'unit' environments
+tox -e fmt             # update your code according to linting rules
+tox -e lint            # code style
+tox -e unit            # unit tests
+tox -e integration-*   # integration tests
```

Comment:
Sure! I thought you know some black magic :-) 
The document was copy&paste compatible previously.

AFAIK, to provide good hint we upgrade to tox 4 (when possible) and use labels: `-m` instead of `-e`.

So far I would use:
```
tox -e integration-charm   # integration test #1
tox -e integration-...     # check all tests in tox.ini

```